chartit/templatetags/chartit.py

```diff
@@ -9,6 +8,14 @@
 from ..charts import Chart, PivotChart
 
 try:
+    from django.utils import simplejson
```

Comment:
Why even try to import `django.utils.simplejson` if the `simplejson` package is required? Just import that directly and then throw an error if it can't be imported.